src/SpectatorView.Unity/Assets/SpectatorView/Scripts/SpatialAlignment/SpatialCoordinateSystemManager.cs

```diff
@@ -265,16 +276,17 @@ private void SendLocalizationCompleteCommand(SocketEndpoint socketEndpoint, bool
 
         private void OnConnected(SocketEndpoint endpoint)
         {
-            if (participants.ContainsKey(endpoint))
+            if (participants.TryGetValue(endpoint.Address, out var existingParticipant))

```

Comment:
>if (participants.TryGetValue(endpoint.Address, out var existingParticipant)) [](start = 12, length = 76)

Why the change to use the address? I'm confused about when this could happen and why it's not an error.

Reply:
reverting